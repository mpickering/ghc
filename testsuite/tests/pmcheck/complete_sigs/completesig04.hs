module TyMismatch where

data A = A
data B = B

{-# COMPLETE A, B #-}
