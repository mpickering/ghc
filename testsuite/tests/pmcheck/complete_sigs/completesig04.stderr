
completesig04.hs:6:1: error:
    • Cannot form a group of complete patterns from patterns ‘A’ and ‘B’ as they match different type constructors (‘A’ resp. ‘B’)
    • In {-# COMPLETE A, B #-}
